sdk/keyvault/azure-security-keyvault-secrets/src/main/java/com/azure/security/keyvault/secrets/SecretClient.java

```diff
@@ -653,7 +772,10 @@ public PagedIterable<SecretProperties> listPropertiesOfSecrets() {
      */
     @ServiceMethod(returns = ReturnType.COLLECTION)
     public PagedIterable<SecretProperties> listPropertiesOfSecrets(Context context) {
-        return implClient.listPropertiesOfSecrets(context);
+        return new PagedIterable<>(maxResults -> mapSecretItemPage(implClient.getSecretsSinglePage(vaultUrl, maxResults,
+            context)),
+            (continuationToken, maxResults) -> mapSecretItemPage(implClient.getSecretsNextSinglePage(continuationToken,
+                vaultUrl, context)));
```

Comment:
I take it we don't map to any specific exception because we didn't use to before?

Reply:
Yup, there were no custom mappings before.